git-tutorial.markdown

```diff
@@ -98,11 +98,77 @@ If you have your old installation under `~/alice` then we suggest you use a
 different directory to avoid confusion.
 
 
+### I don't use aliBuild
+
+If you do not use aliBuild you can keep using your previous installation
+commands without any modification. You need however to clone the ALICE
+repositories from GitHub:
+
+```bash
+cd ~/alice
+git clone https://github.com/alisw/AliRoot
+git clone https://github.com/alisw/AliPhysics
+```
+
+
+### I have participated to the GitHub test
+
+> These instructions are only valid from the time the GitHub migration has been
+> completed!
+
+If you have participated to the GitHub test and you want to sync with the
+current official AliRoot/AliPhysics and alidist repositories from upstream,
+starting from the day when Git migration is effective you can follow this
+procedure and you will save some build time.
+
+Assuming you have your GitHub test installation under `~/alice`, update your
+recipes:
+
+```bash
+cd ~/alice
+rm -rf alidist/
```

Comment:
Because we change branch (`git-migration` becomes `master`), and because we will change alidist's _default_ branch (and `git clone` without `-b` will do The Right Thing). I am not aware of a simpler method to do that.